### ISSUE TITLE
Enhancement: [SITE appears to be offline or unresponsive] Notifications triggered by plugin updates

### ISSUE THREAD

**Issue by kimclow** [Automattic/jetpack#27735]
### Impacted plugin

Jetpack, Backup, VaultPress

### What

We receive "SITE appears to be offline or unresponsive notification" during plugin updates. 

### How

Hi there! 
I work in a team with a couple of hundred Jetpack sites on our WP.com user account. 

We've found that sometimes when a site is doing its plugin updates, we'll get the notification that it is offline or responsive. 
[Alert: SITE appears to be offline or unresponsive]

Due to the high volume of sites we have, this can happen frequently. It can be disruptive as each time it happens, we check the site/logs/Jetpack etc., to see why we received the notification. 

This is just a small request to see if there is something we can improve on the functionality of how Jetpack understands plugin updates. 

I don't know how Jetpack functions, so I cannot provide any recommendations here, however I just thought I would share the issue and see if any JP wizards have any ideas. 

Thank you so much!

**Comment by kimclow** [comment]
I misunderstood the root of this issue. I had thought it was a Jetpack issue but it seems to be in core rather. I'm closing this. 

TL;DR what's happening is that the WP Core plugin auto-updater runs, and updates the plugin, during the plugin update operation a request tries to execute a code path that triggers this WP Core error alert.
